### PR TITLE
Sb 14793

### PR DESCRIPTION
**Pull request by manzarul**

### CODE REVIEW

File: samza/notification-job/src/main/config/notification-job.properties [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# Job Coordinator
+job.coordinator.system=kafka
+
+# Normally, this would be 3, but we have only one broker.
+job.coordinator.replication.factor=1
+
+# Job specific configuration
+fcm_account_key==__fcm_account_key__
+user_cert_merge_private_api=/private/user/v1/certs/merge
```
Comment (Hari-stackroute) [inline samza/notification-job/src/main/config/notification-job.properties:46]: Please remove unnecessary key-value pairs

File: samza/notification-job/src/main/java/org/sunbird/jobs/samza/util/Constant.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	public static final String ACTOR = "actor";
+	public static final String ID = "id";
+	public static final String TYPE  = "type";
+	public static final String EID = "eid";
+	public static final String EDATA = "edata";
+	public static final String ACTION = "action";
+	public static final String REQUEST = "request";
+	public static final String NOTIFICATION = "notification";
+    public static final String MODE = "mode";
+    public static final String DELICERY_TYPE = "deliveryType";
```
Comment (Hari-stackroute) [inline samza/notification-job/src/main/java/org/sunbird/jobs/samza/util/Constant.java:18]: Please check spell

File: samza/notification-job/src/main/java/org/sunbird/jobs/samza/task/NotificationJob.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+/**
+ * 
+ * @author manzarul
+ *
+ */
+public class NotificationJob implements StreamTask, InitableTask {
+
+    private static NotificationService service = new NotificationService();
+    private JobLogger Logger = new JobLogger(NotificationService.class);
```
Comment (Hari-stackroute) [inline samza/notification-job/src/main/java/org/sunbird/jobs/samza/task/NotificationJob.java:27]: Should be NotificationJob.class

File: samza/notification-job/src/main/java/org/sunbird/jobs/samza/task/NotificationJob.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        }
+    }
+
+    @SuppressWarnings("unchecked")
+    private Map<String, Object> getMessage(IncomingMessageEnvelope envelope) {
+        try {
+            return (Map<String, Object>) envelope.getMessage();
+        } catch (Exception e) {
+            Logger.error("NotificationJob:getMessage: Invalid message = " + envelope.getMessage(), e);
+            return new HashMap();
```
Comment (Hari-stackroute) [inline samza/notification-job/src/main/java/org/sunbird/jobs/samza/task/NotificationJob.java]: What is this HashpMap() is returned? there is a possibility of null pointer exception!

File: samza/notification-job/src/main/java/org/sunbird/jobs/samza/task/NotificationJob.java (hunk 3 above)
Comment (manzarul) [reply samza/notification-job/src/main/java/org/sunbird/jobs/samza/task/NotificationJob.java:27]: Done in separate PR

File: samza/notification-job/src/main/config/notification-job.properties (hunk 1 above)
Comment (manzarul) [reply samza/notification-job/src/main/config/notification-job.properties:46]: Done in separate PR

**Pull request merged**